js/rank_states_utils.js

```diff
@@ -209,11 +218,29 @@ function rankEm(barData) {
       .attr('text-anchor','middle')
       .text('Drag a state over its matching bar');
   
-  svgStates.append('g')
+  var labelTextGroup = svgStates.append('g')
     .attr('id','rank-data-text')
-    .attr('transform',"translate("+(rankSvg.width * 0.4)+","+(rankSvg.height * 0.42)+")")
-    .append('text')
-      .text(' ');
+    .attr('text-anchor','middle')
+    .attr('display', 'none');
+  
+  labelTextGroup.append("text")
```

Comment:
if all of these are being bumped the same way (x = width.0.6 etc), you can put it in as the transform=translate(width0.6,height0.3) and remove the x and y attributes from the child elements that follow (default in svg is x and y = 0).